server/tests/model/test_user.py

```diff
@@ -100,12 +99,9 @@ def test_get_id(mocker: MockerFixture) -> None:
 def test_unassigned_family_id(mocker: MockerFixture) -> None:
     """Test unassigned_family_id was created and populated."""
     user = add_mock_user(mocker, userAlbert)
-    family_id = user.unassigned_family_id
-    assert family_id
-
-    family = Family.lookup_family(family_id)
+    family = user.unassigned_family_id.fetch()
```

Comment:
can we wrap this call in a method so it doesnt look like you're plopping an id into a family variable?

Reply:
user.get_unassigned_family()
or
a @property decorated
user.unassigned_family

Reply:
good idea, I'll refactor